tests/serverpod_test_server/lib/src/models/object_with_postgis.spy.yaml

```
@@ -0,0 +1,25 @@
+class: ObjectWithPostgis
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)